localnet/path/config/shannon.example.yaml

```diff
@@ -0,0 +1,22 @@
+# To run a Local development environment using Shannon as the relaying protocol, copy this file and rename it to .config.yaml, then update
+# the `protocol.shannon_config.full_node_config.gateway_private_key` value with the correct value.
+# NOTE: only the gateway_private_key value should be updated.
+# shannon_config is required for using the shannon relayer in the Localnet environment.
+shannon_config:
+  full_node_config:
+    # do not change rpc_url
+    rpc_url: https://shannon-testnet-grove-seed-rpc.poktroll.com 
+    grpc_config:
+      # do not change host_port
+      host_port: shannon-testnet-grove-seed-grpc.poktroll.com:443 
+    # NOTE: gateway_private_key must be replaced with the correct gateway private key secret
+    gateway_private_key: cf09805c952fa999e9a63a9f434147b0a5abfd10f268879694c6b5a70e1ae177 
```

Comment:
Where do I go (link?) to learn more about generate a new gateway_private_key?

This is the first question I have even as an experienced developer.

Reply:
Thanks, added a link to the Shannon's docker compose walkthrough docs.